src/cfn/template.yaml

```
@@ -4,7 +4,7 @@ Transform: AWS::Serverless-2016-10-31
 
 Globals:
   Function:
-    Runtime: nodejs12.x
+    Runtime: nodejs14.x
```

we just released the 16 runtime, can we do that instead?
```suggestion
    Runtime: nodejs16.x
```